app/grandchallenge/archives/views.py

```diff
@@ -696,3 +702,97 @@ def get_serializer_class(self):
             return ArchiveItemPostSerializer
         else:
             return ArchiveItemSerializer
+
+
+class ArchiveItemCreateView(
+    ObjectCreateMixin,
+    MultipleCIVProcessingBaseView,
+):
+    form_class = MultipleCIVForm
+    permission_required = (
+        f"{Archive._meta.app_label}.change_{Archive._meta.model_name}"
+    )
+    included_form_classes = (
+        MultipleCIVForm,
+        *MultipleCIVProcessingBaseView.included_form_classes,
+    )
+    success_message = "Archive item has been created."
+    type_to_add = ArchiveItem._meta.verbose_name
+    base_object_type = Archive._meta.model_name
+
+    def get_permission_object(self):
+        return self.base_object
+
+    @property
+    def base_object(self):
+        return Archive.objects.filter(slug=self.kwargs["slug"]).get()
+
+    @property
+    def list_url(self):
+        return reverse("archives:list")
+
+    @property
+    def form_url(self):
+        return reverse(
+            "archives:item-create", kwargs={"slug": self.base_object.slug}
+        )
+
+    @property
+    def return_url(self):
+        return reverse(
+            "archives:items-list", kwargs={"slug": self.base_object.slug}
+        )
+
+    @property
+    def new_interface_url(self):
+        return reverse(
+            "archives:item-new-interface-create",
+            kwargs={"slug": self.base_object.slug},
+        )
+
+    def process_data_for_object(self, data):
+        """Creates an archive item"""
+        instance = ArchiveItem.objects.create(archive=self.base_object)
+        for slug in data:
+            instance.create_civ(
+                ci_slug=slug, new_value=data[slug], user=self.request.user
+            )
+        return instance
+
+    def get_success_url(self):
+        return self.return_url
+
+
+class ArchiveItemInterfaceCreate(InterfacesCreateBaseView):
+    permission_required = (
+        f"{Archive._meta.app_label}.change_{ArchiveItem._meta.model_name}"
+    )
+
+    def get_permission_object(self):
+        return self.object
```

Comment:
`self.object` is `None` when the pk does not exist? How does the permission check work there?

Reply:
Very good question. I had just taken this bit over from Mike's code without questioning it since everything worked as expected, but looking into it now, the answer is that the permission check will pass if the permission object is None. That is strange,, but lokks like it is intended to work this way in Django guardian.

For us, I'm actually thinking now that we don't need to protect this view with object permissions since it's just a template view that is used within the UpdateView where we check permissions. Do you agree?

Reply:
My guess is that by passing `None` for the object you then do a check against the global permission. I think it should still have permissions attached as you're getting information about an object. Is this the same for a reader study view?

Reply:
Yes, it's the same for the reader study view. Ok, I'll add the permission check again. 

It does not check global permissions because we have `accept_global_perms` set to False. `check_permissions` simply returns `None` and if that happens the view is executed.

Reply:
Yes, please ensure permissions are checked in both places and ensure there is a test for this.